integration-tests/src/test/resources/features/source-slack.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Slack Source tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+
+  @slacksource
+  Scenario: Slack Source Processor is created and slack message as source should result into other generated slack message
+    When add a Processor to the Bridge "mybridge" with body:
+    """
+    {
+      "name": "slackSourceProcessor",
+      "source": {
+        "type": "Slack",
+        "parameters": {
+            "channel": "${env.slack.channel.name}",
+            "token": "${env.slack.webhook.token}"
+          }
+      },
+      "filters": [
+        {
+          "key": "data.text",
+          "type": "StringBeginsWith",
+          "values": ["Slack Event Source Feature trigger"]
+        }
+      ]
+    }
+    """
+    And the Processor "slackSourceProcessor" of the Bridge "mybridge" is existing with status "ready" within 5 minutes
+    And the Processor "slackSourceProcessor" of the Bridge "mybridge" has source of type "Slack"
+
+    When add a Processor to the Bridge "mybridge" with body:
```

Comment:
Nitpick, if you move creating `slackForwardProcessor` right after creating `slackSourceProcessor` and then you check availability of both, you can save significant amount of test time (as both Processors will start in parallel).

Reply:
done